backend/test/mastodon/oauth.spec.ts

```diff
@@ -126,15 +127,15 @@ describe('Mastodon APIs', () => {
 			const location = res.headers.get('location')
 			assert.equal(location, 'https://redirect.com/a')
 
-			const actor = await db.prepare('SELECT * FROM actors').first()
+			const actor = await db.prepare('SELECT * FROM actors').first<{ properties: string; email: string; id: string }>()
```

Comment:
How about 

```
   .first<{ properties: string; email: string; id: string }&Actor>()
```

then we don't need the `as unknown as Actor` below.

Reply:
ah yeah totally 🙂